python/EBS_SNAPSHOT_PUBLIC_RESTORABLE_CHECK/EBS_SNAPSHOT_PUBLIC_RESTORABLE_CHECK_test.py

```diff
@@ -0,0 +1,269 @@
+                                     'server': 'AmazonEC2'}, 'HTTPStatusCode': 200, 'RequestId': 'example09-ecb6-407e-9053-e8sample5f',
+                                     'RetryAttempts': 0},'Snapshots': []}
+        ec2_client_mock.describe_snapshots = MagicMock(return_value=describe_snapshots_result)
+        lambda_result = rule.lambda_handler(self.lambda_event, {})
+        expected_response = [build_expected_response(compliance_type='NOT_APPLICABLE',
+                                                    compliance_resource_id='N/A', compliance_resource_type=DEFAULT_RESOURCE_TYPE)]
+        assert_successful_evaluation(self, lambda_result, expected_response, len(lambda_result))
+
+# Checks for scenario wherein API call returns an error
+class APIErrorTest(unittest.TestCase):
+    lambda_event = {}
+
+    def setUp(self):
+        self.lambda_event = build_lambda_scheduled_event()
+        pass
+
+    def test_api_error(self):
+        error_response = {'ResponseMetadata': {'HTTPHeaders': {'content-type': 'text/xml;charset=UTF-8',
+                                                                'date': 'Fri, 15 Mar 2019 16:55:26 GMT',
+                                                                'server': 'AmazonEC2',
+                                                                'transfer-encoding': 'chunked',
+                                                                'vary': 'Accept-Encoding'},
+                                                                'HTTPStatusCode': 403,
+                           'RequestId': 'fa2f8132-c8eb-46c3-b242-4example42a',
+                           'RetryAttempts': 0},
+                           'Snapshots': [{'Description': 'Copied for DestinationAmi ami.',
+                                        'Encrypted': False,
+                                        'OwnerId': '123456789012',
+                                        'Progress': '100%',
+                                        'SnapshotId': 'snap-0daeb11514fba831a',
+                                        'StartTime': datetime.datetime(2019, 3, 15, 14, 58, 48, 662000, tzinfo=tzutc()),
+                                        'State': 'completed',
+                                        'VolumeId': 'vol-ffffffff',
+                                        'VolumeSize': 99}]}
+        ec2_client_mock.describe_snapshots = MagicMock(return_value=error_response)
+        lambda_result = rule.lambda_handler(self.lambda_event, {})
+        expected_result = []
+        self.assertEqual(expected_result, lambda_result)
+
+    #def test_sample_2(self):
+    #    rule.ASSUME_ROLE_MODE = False
+    #    response = rule.lambda_handler(build_lambda_configurationchange_event(self.invoking_event_iam_role_sample, self.rule_parameters), {})
+    #    resp_expected = []
+    #    resp_expected.append(build_expected_response('NOT_APPLICABLE', 'some-resource-id', 'AWS::IAM::Role'))
+    #    assert_successful_evaluation(self, response, resp_expected)
```

Comment:
You can remove these un-necessary comments.

Reply:
Removed the commented lines